fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/PermissionsRepository.java

```diff
@@ -50,6 +51,16 @@
   Map<String, UserPermission> getAllById();
 
   /**
+   * Gets all UserPermissions in the repository that has at least 1 of the specified roles, keyed
+   * by user ID. If anyRoles is null, returns the same result as getAllById() (which
+   * includes the unrestricted user). If anyRoles is empty, it returns a map with a single entry for
+   * the unrestricted user.
```

Comment:
Can you explain why this is? I.e. "If `anyRoles` is non-empty, then by definition the unrestricted user is excluded... etc".